import exceptions

class MapperError(exceptions.Exception):
	"""All Exceptions thrown in the Mapper Relam of things should be this. It is a stub that maybe filled out later"""
	pass